src/server/views/payment/cash.njk

```diff
@@ -0,0 +1,68 @@
+{% extends 'layouts/default.layout.njk' %}
+
+{% set pageTitle = 'DVSA Road Side Payment Portal' %}
+{% set pageBreadcrumbItems = [
+    { text: 'Home', url: '/' },
+    { text: 'Payment Code', url: '/' },
+    { text: 'Penalty Details', url: '/payment-code/'+paymentCode },
```

Comment:
Maybe use `/payment-code/${paymentCode}`?

Reply:
I've checked nunjucks documentation and unfortunately we cannot use javascript-style template literals  :(